src/main/resources/story/walmart/CreateUser.story

```diff
@@ -0,0 +1,16 @@
+Description: Story to test user creation;
+
+Scenario: Open Walmart main page
+Given I am on a page with the URL 'https://www.walmart.com/'
+Then the page title is 'Walmart.com | Save Money. Live Better.'
+
+Scenario:  Create user
+When I click on an element by the xpath '//button[@aria-label='Your Account']'
+When I click on an element by the xpath  '//div[contains(text(),'Create Account')]' 
+When I enter '#{generate(Name.firstName)}' in a field by the xpath '//input[@id='first-name-su']'
```

Comment:
This xpath is ok, but alternatively you can refer to attribute @name or @placeholder which are more user friendly

Reply:
Fixed for 'First name' and 'Last name'. Email and Password have not unique @name/@placeholder - go with xpath for them.